### DISCUSSION TITLE
로그인, 회원가입 웹페이지 디자인 수정중인데 Header 여부 관련 논의

### DISCUSSION THREAD

**Discussion by JangGusWjd** [MajorIn-ABO/MajorIn_FE#14] [Q&A]
<img width="1470" alt="스크린샷 2023-11-06 오후 10 42 48" src="https://github.com/MajorIn-ABO/MajorIn_FE/assets/88546743/76297898-de9b-42d1-8714-226a1ec69db1">

로그인이랑 회원가입 웹페이지 디자인 위에 처럼 수정했는데 Header 부분 제외할까 아니면 그냥 둘까?

**Comment by cho-ja-young** [comment]
로그인 페이지는 네이버 로그인 페이지처럼 위의 헤더 없애는게 좋을 것 같아!! 로고가 두번 들어가면 로그인으로 몰입이 안 되는 느낌이 좀 들어서!!
회원가입 페이지는 헤더 있는게 좋을 것 같아!! 지금 그대로!!

전체적으로 다 깔끔하고 예쁘당 >ㅁ<

**Reply by wlsl10** [reply]
로그인은 내용이 적어 로고가 들어가서 헤드의 로고와 중복되서 빼면 좋을 것 같아!
굳굳!
